jellyfin_apiclient_python/api.py

```diff
@@ -343,13 +343,16 @@ def get_recently_added(self, media=None, parent_id=None, limit=20):
             'Fields': info()
         })
 
-    def get_next(self, index=None, limit=1):
-        return self.shows("/NextUp", {
-            'Limit': limit,
-            'UserId': "{UserId}",
-            'StartIndex': None if index is None else int(index)
-        })
-
+    def get_next(self, userId=None, index=None, limit=1, seriesId=None):
```

Comment:
To ensure backwards compatibility please do not add new arguments before existing ones.